x/incentives/keeper/distribute.go

```diff
@@ -379,20 +381,23 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 
 	for _, gauge := range gauges {
 		var gaugeDistributedCoins sdk.Coins
-		pool, isCLPool := k.GetValidConcentratedLiquidityGauge(ctx, gauge.Id, currentEpoch.Duration)
-		// only want to run this logic if the gaugeId is associated with CL PoolId
-		if isCLPool {
+		pool, err := k.GetPoolFromGauge(ctx, gauge.Id, currentEpoch.Duration)
+		// Note: getting NoPoolAssociatedWithGaugeError implies that there is no pool associated with the gauge but we still want to distribute to base locks.
+		if err != nil && !errors.Is(err, poolincentivestypes.NoPoolAssociatedWithGaugeError{GaugeId: gauge.Id, Duration: currentEpoch.Duration}) {
```

Comment:
what if there is an error here, where a pool is associated with a gauge but it fails to getPool. For example: 
`GetPoolIdFromGaugeId` passes and `GetPool` fails

Reply:
aah i see, in that case pool would nil and we'll run the base case right?